src/main/scala/uk/gov/hmrc/ct/box/ValidatableBox.scala

```diff
@@ -227,4 +227,7 @@ trait ValidatableBox[T <: BoxRetriever] {
     }
   }
 
+  def validateDeclarationNameOrStatus(boxId: String, box: CtOptionalString): Set[CtValidation] = {
```

Comment:
I would pull this out into it's own trait as its specific validation and not generic, like the Registered Charity Number